### ISSUE TITLE
Unknown option -f

### ISSUE THREAD

**Issue by tetricky** [efrecon/sqlite-backup#5]
I've tried to use the example you provide, for dumping an sqlite database.
```
docker run -it --rm \
  -v $(cwd):/data \
  efrecon/sqlite-backup \
    -f /data/db.sqlite3 \
    -k 10 \
    -d /tmp \
    -t cat
```
If I do this, I get "Unknown option -f"

How do I point the container at the source database correctly?

**Comment by efrecon** [comment]
I should update the README... :) It now takes a list of files after all options instead. So instead of `-f xxx` you place the path to the file after all options, `-- xxx` (the `--` ensure this is the end of options and beginning or arguments, just in case your filename started with a dash). Running the script with `-h` should have printed a usage description that says what I have just written here.

**Comment by efrecon** [comment]
Fixed by a703fecb25b3409590bb7d6ee53cc38de1a2497e
